kolibri/plugins/coach/assets/src/views/new/shared/status/InProgress.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+
+  <LabeledIcon :label="text">
+    <mat-svg category="device" name="access_time" />
+  </LabeledIcon>
+
+</template>
+
+
+<script>
+
+  import statusMixin from './status';
+
+  export default {
+    name: 'InProgress',
+    mixins: [statusMixin],
+    $trs: {
+      basicLabelInProgress: '{count, plural, other {In-progress}}',
+      countShortInProgress: '{count, number, integer}',
+      ratioShortInProgress: '{count, number, integer} of {total, number, integer}',
+      countInProgress: '{count, number, integer} {count, plural, other {in-progress}}',
+      ratioInProgress:
+        '{count, number, integer} of {total, number, integer} {count, plural, other {in-progress}}',
```

Comment:
Same here and below, I see no need for a hyphen.

Reply:
addressed - 558dbbc0b59e95340fc7ae427ee3b79e7580f0b8